src/manage_sql_secinfo.c

```diff
@@ -2786,35 +2787,35 @@ insert_cve_from_entry (element_t entry, element_t last_modified,
 
   if (base_metrics == NULL)
     {
-      score = NULL;
+      score = 0;
       cvss_vector = NULL;
     }
   else
     {
-      score = element_child (base_metrics,
-                             cvss_is_v3 ? "cvss3:base-score" : "cvss:score");
-      if (score == NULL)
-        {
-          g_warning ("%s: cvss:score missing", __func__);
-          g_free (id);
-          return -1;
-        }
+      double base;
 
       cvss_vector = element_child (base_metrics,
                                    cvss_is_v3 ? "cvss3:vector-string"
                                               : "cvss:vector-string");
+      if (cvss_is_v3 && (cvss_vector == NULL))
+        /* At least nvdcve-2.0-2004.xml uses this format. */
```

Comment:
This is actually about an upcoming change in the feed that will affect all CVE XML files, not just the 2004 one.